### ISSUE TITLE
Tooltip of close button appears on ToastNotification icon

### ISSUE THREAD

**Issue by brunnerh** [carbon-design-system/carbon-components-svelte#672]
![image](https://user-images.githubusercontent.com/834235/121032919-04935980-c7ac-11eb-9176-78f1583febfa.png)

Reason: Same `iconDescription` variable is used for both icons

[Source](https://github.com/IBM/carbon-components-svelte/blob/master/src/Notification/ToastNotification.svelte):

```svelte
<NotificationIcon kind="{kind}" iconDescription="{iconDescription}" />
<NotificationTextDetails
  title="{title}"
  subtitle="{subtitle}"
  caption="{caption}"
>
  <slot />
</NotificationTextDetails>
{#if !hideCloseButton}
  <NotificationButton
    iconDescription="{iconDescription}"
    on:click="{close}"
  />
{/if}
```

**Comment by metonym** [comment]
Fixed in v0.37.0

**Comment by brunnerh** [comment]
@metonym I do not know if i properly tested the fix at the time, but the problem appears to exist right now because [`NotificationIcon` has a default `iconDescription`](https://github.com/carbon-design-system/carbon-components-svelte/blob/master/src/Notification/NotificationIcon.svelte)

```js
export let iconDescription = "Closes notification";
```


![image](https://user-images.githubusercontent.com/834235/139587923-923531d5-536b-42df-b683-4957edabcb7d.png)

**Comment by metonym** [comment]
I'll take another look.

**Comment by theetrain** [comment]
Released in [v0.71.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.71.0)
